packages/composer/test/graphql/watch.test.js

```diff
@@ -55,15 +55,10 @@ test('should restart composer if a service has been changed, and update the sche
     },
   })
 
-  assert.equal(composer.graphqlSupergraph.sdl, schema1)
-
   await graphql1.close()
   await graphql1a.listen({ port })
 
-  await restart
-
-  assert.equal(composer.restarted, true, 'expected composer to restart')
-  assert.equal(composer.graphqlSupergraph.sdl, schema2)
```

Comment:
this assertion is important

Reply:
I restored in a different way but I now check schemas.